source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputChoiceSetRenderer.mm

```diff
@@ -36,7 +36,13 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
     std::shared_ptr<ChoiceSetInput> choiceSet = std::dynamic_pointer_cast<ChoiceSetInput>(elem);
     // creates a tableview with pre-defined style
-    ACRInputTableView *choiceSetView = [[ACRInputTableView alloc] initWithSuperview:viewGroup];
+    NSBundle *bundle = [NSBundle bundleWithIdentifier:@"MSFT.AdaptiveCards"];
+    if(!bundle){
+        // TODO Add Render Error - jwoo
```

Comment:
>// TODO Add Render Error - jwoo [](start = 8, length = 31)

Is there a bug open to track this work? It would be a good idea to put a reference to the open Github issue here in the comment.

Reply:
thanks; will do;
[https://github.com/Microsoft/AdaptiveCards/issues/1834](https://github.com/Microsoft/AdaptiveCards/issues/1834)

---
In reply to: [213129615](https://github.com/Microsoft/AdaptiveCards/pull/1830#discussion_r213129615) [](ancestors = 213129615)

Reply:
I meant to put the reference in the comment in the code. It just helps to avoid the code getting littered with TODOs that will never be cleaned up.

---
In reply to: [213148522](https://github.com/Microsoft/AdaptiveCards/pull/1830#discussion_r213148522) [](ancestors = 213148522,213129615)

Reply:
i see; thanks

---
In reply to: [213149076](https://github.com/Microsoft/AdaptiveCards/pull/1830#discussion_r213149076) [](ancestors = 213149076,213148522,213129615)